clusterloader2/pkg/measurement/util/wait_for_pvcs.go

```diff
@@ -0,0 +1,92 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/util/sets"
+	clientset "k8s.io/client-go/kubernetes"
+	"k8s.io/klog"
+)
+
+// WaitForPVCOptions is an options used by WaitForPVCs methods.
+type WaitForPVCOptions struct {
+	Selector            *ObjectSelector
+	DesiredPVCCount     int
+	EnableLogging       bool
+	CallerName          string
+	WaitForPVCsInterval time.Duration
+}
+
+// WaitForPVCs waits till disire nuber of PVCs is running.
+// PVCs are be specified by namespace, field and/or label selectors.
+// If stopCh is closed before all PVCs are running, the error will be returned.
+func WaitForPVCs(clientSet clientset.Interface, stopCh <-chan struct{}, options *WaitForPVCOptions) error {
+	ps, err := NewPVCStore(clientSet, options.Selector)
+	if err != nil {
+		return fmt.Errorf("PVC store creation error: %v", err)
+	}
+	defer ps.Stop()
+
+	var pvcsStatus PVCStartupStatus
+	scaling := uninitialized
```

Comment:
what is `uninitialized`? Why is this not a variable declaration i.e. `var scaling *Type`

Reply:
The same as above (and I won't comment on other).
This is not the problem with this PR, but the fact that it was already done like this before. I think consistency is more important that cleaning this up in just one place.

@krzysied - btw, those are valid comments that we may consider applying

Reply:
ok, this can be a separate PR then